src/env.ts

```diff
@@ -3,31 +3,45 @@ export interface Env {
    * Environment the worker is running in
    */
   ENVIRONMENT: 'dev' | 'staging' | 'prod' | 'e2e-tests';
+
+  /**
+   * Should caching be enabled?
+   */
+  CACHING: boolean;
+
+  LOG_ERRORS?: boolean;
```

Comment:
I forget what the actual number is but there is a limit on the about of bindings & vars a worker can have, so might be beneficial to have these in a `FLAGS` object or something